gqcp/tests/ONVBasis/OperatorString_test.cpp

```diff
@@ -89,6 +125,65 @@ BOOST_AUTO_TEST_CASE(test_unresolved_isZero) {
     BOOST_CHECK_EQUAL(operator_string_2.isZero(), true);
 }
 
+
```

Comment:
A description of the test is missing here. I would add it, but it's finicky to get the right indentation on here.